package.json

```diff
@@ -25,8 +31,9 @@
     "cross-env": "^7.0.3",
     "dotenv": "^16.4.5",
     "express": "^4.18.2",
-    "knex": "^3.0.1",
-    "mysql2": "^3.6.1"
+    "knex": "^3.1.0",
+    "mysql2": "^3.6.1",
+    "nodemon": "^3.1.0"
```

Comment:
I already have moved nodemon in dev mode, if you run your server with `npm run dev` instead of `run start` it shouldn't readd it to prod dependencies.